lms/validation/__init__.py

```diff
@@ -0,0 +1,11 @@
+from lms.validation._exceptions import ValidationError
+from lms.validation._parser import parser
+from lms.validation._oauth import CANVAS_OAUTH_CALLBACK_ARGS
+
+
+__all__ = ("parser", "CANVAS_OAUTH_CALLBACK_ARGS", "ValidationError")
+
+
+@parser.error_handler
+def _handle_error(error, _req, _schema, _status_code, _headers):
+    raise ValidationError(messages=error.messages) from error
```

Comment:
webargs calls this function whenever schema validation fails. It just wraps the exception from webargs in a custom `ValidationError` class. Without this function webargs's Pyramid integration would just cause a validation failure to raise `pyramid.httpexceptions.HTTPUnprocessableEntity`. We'll see the need for a custom exception class (other than simply encapsulating webargs) below.